src/ctia/http/routes/exploit_target.clj

```diff
@@ -69,16 +74,19 @@
                 :header-params [api_key :- (s/maybe s/Str)]
                 :summary "List ExploitTarget by external id"
                 :capabilities #{:read-exploit-target :external-id}
+                :middleware [wrap-un-store wrap-cache-control]
                 (paginated-ok
                  (page-with-long-id
                   (read-store :exploit-target list-exploit-targets
                               {:external_ids external_id} q))))
+
            (GET "/search" []
                 :return (s/maybe [StoredExploitTarget])
```

Comment:
well I think this is cleaner, otherwise we should remove the :return param of the route as it is wrong.